html_css/v2/main-course/intermediate_css/frameworks-preprocessors.md

```diff
@@ -0,0 +1,91 @@
+### Introduction
+
+At this point, you have written quite a bit of vanilla HTML and CSS, and learned many design techniques that you will continue to use as you grow as a developer. However, through your experiences, you have no doubt encountered a CSS framework or two. You may have also learned about preprocessors (aka precompilers). Both of these types of tools can make writing CSS more streamlined and less tedious.
+
+You should be aware that at this point in your learning, both frameworks and preprocessors are options for you, but neither of them are necessary. In particular, you'll read a strong argument below AGAINST using frameworks at this point.
```

Comment:
in a later comment I suggest removing the strong "DO NOT USE" statement... so i'd reword this a bit.

maybe say something about trade-offs etc.

also.. the real reason students need to know this stuff is that they're pretty commonly found in workplaces.  Many job requirements list bootstrap or SASS or something.. so it's helpful for students to at least know what they are and where to look once they've determined they need to learn them.

Reply:
Got it